src/autoscaler/pruner/cmd/pruner/pruner_test.go

```diff
@@ -73,9 +73,19 @@ var _ = Describe("Pruner", func() {
 				}))
 		})
 
-		It("should start", func() {
```

Comment:
Yes can separate test for .Should(Say("pruner.started")) but still need eachConsistently(runner.Session).ShouldNot(Exit()) in each test isn't it